subsys/bluetooth/controller/Kconfig

```diff
@@ -38,6 +38,7 @@ config BT_LL_SOFTDEVICE
 	select BT_HAS_HCI_VS
 	select BT_CTLR_DF_SUPPORT if HAS_HW_NRF_RADIO_DFE
 	select BT_CTLR_LE_POWER_CONTROL_SUPPORT
+	select BT_CTLR_SUBRATING_SUPPORT
```

Comment:
You will need to merge this kconfig to zephyr before merging here, also I don't think we should select it as it is not currently supported even in an experimental state

Reply:
It is necessary to select subrating support here